modules/cnf-gathering-data-about-cluster-using-must-gather.adoc

```diff
@@ -22,20 +22,20 @@ The Performance Profile Creator (PPC) tool requires `must-gather` data. As a clu
 +
 [source,terminal]
 ----
-$ oc adm must-gather --image=<PAO_image> --dest-dir=<dir>
+$ oc adm must-gather --image=<NTO_image> --dest-dir=<dir>
 ----
 +
 [NOTE]
 ====
-`must-gather` must be run with the `performance-addon-operator-must-gather` image. The output can optionally be compressed. Compressed output is required if you are running the Performance Profile Creator wrapper script.
+`must-gather` must be run with the `node-tuning-operator-must-gather` image. The output can optionally be compressed. Compressed output is required if you are running the Performance Profile Creator wrapper script.
```

Comment:
As we are still using performance-addon-operator image to run must-gather maybe this one should be left unchanged.